app/controllers/movie_controller.rb

```diff
@@ -0,0 +1,40 @@
+class MovieController < ApplicationController
+  def top_forty
```

Comment:
I am wondering if (AFTER THE REFACTOR), this ends up being an index action that routes to EITHER the top40 results or the search results.

I can also see the we would add a show action that would be the movie details page :)